src/schema/users.ts

```diff
@@ -1320,6 +1377,22 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       return personalizedDigest;
     },
+    companies: async (_, __, ctx: AuthContext, info) => {
+      const userCompanies = await graphorm.query(ctx, info, (builder) => {
+        builder.queryBuilder = builder.queryBuilder
+          .andWhere(`${builder.alias}."userId" = :userId`, {
+            userId: ctx.userId,
+          })
+          .andWhere(`${builder.alias}."verified" = true`);
+
+        return builder;
+      });
+      if (userCompanies.length === 0) {
+        throw new NotFoundError('No companies found');
```

Comment:
Do we need to throw error, or just return empty array or null?

Reply:
Ah good point, empty array is fine :)
adjusted (Also are you not on leave?)

Reply:
> (Also are you not on leave?)
🙈